itou/www/job_seekers_views/views.py

```diff
@@ -51,6 +52,23 @@
 logger = logging.getLogger(__name__)
 
 
+def jobseeker_personal_infos_display_kwargs(jobseeker_profile):
+    other_infos_keys = [
+        "resourceless",
+        "unemployed_since",
+        "rqth_employee",
+        "oeth_employee",
+        "rsa_allocation_since",
+        "ass_allocation_since",
+        "aah_allocation_since",
+    ]
+    return {
+        "show_birth_place": not jobseeker_profile.birth_country
+        or jobseeker_profile.birth_country_id == Country.FRANCE_ID,
```

Comment:
```suggestion
        "show_birth_place": jobseeker_profile.birth_country_id == Country.FRANCE_ID,
```
devrait suffire a priori

Reply:
Fonctionnellement, c'est ok.
Dans le cas d'un pays de naissance vide, la ligne commune de naissance n'est pas affichée.